src/main/java/com/softserve/actent/controller/EquipmentController.java

```diff
@@ -64,17 +64,17 @@ public EquipmentController(EquipmentServiceImpl equipmentServiceImpl, UserReposi
         Equipment newEquipment = modelMapper.map(createEquipmentDto, Equipment.class);
 
         // TODO: use User and Event services for check if they exists
-        User user = userRepository.getOne(createEquipmentDto.getUserid());
-        Event event = eventRepository.getOne(createEquipmentDto.getEventid());
+        User user = userRepository.getOne(createEquipmentDto.getUserId());
+        Event event = eventRepository.getOne(createEquipmentDto.getUserId());
```

Comment:
you're fetching event by userId? Could the user have more than 1 event assigned?

Reply:
damn. thanks)